### ISSUE TITLE
Add account/settings icon to Tabula homepage

### ISSUE THREAD

**Issue by eccogrinder** [gnosis/tabula#259]
**Is your feature request related to a problem? Please describe.**
It can be easy to get lost in Tabula. For instance, once you connect your wallet and navigate back to Tabula's homepage, there is no clear way to navigate to your own account. 

**Describe the solution you'd like**
I suggest adding an account icon at the top right (to replace where "Connect Wallet" was previously), so users can easily navigate to their publications.

**Comment by auryn-macmillan** [comment]
There's an interesting tension on tabula because we've tried, as much as possible, to make a publication-centric UX, rather than user-centric.

This means that each publication, once landed on, should feel like the top-level of the application.

That said, I do think there is value in giving users an overview of the publications they have created or have permissions in, along with an easy way to navigate to them. So I generally support this feature suggestion, @eccogrinder.

I'd also add that it would be great to have a view that exposes all of the publications in which the logged in user has permissions, across all networks. As this view is currently specific to the network currently exposed by the user's wallet.

**Comment by eccogrinder** [comment]
> There's an interesting tension on tabula because we've tried, as much as possible, to make a publication-centric UX, rather than user-centric. This means that each publication, once landed on, should feel like the top-level of the application.
> 

Yeah, definitely, and I like how we focus on publications over users. It's just been a bit frustrating on a personal level to find where my "home' is, so to speak. Perhaps there's another way to solve.

> I'd also add that it would be great to have a view that exposes all of the publications in which the logged in user has permissions, across all networks. As this view is currently specific to the network currently exposed by the user's wallet.

100% agree, especially as folk get better accustomed to navigating multiple chains.
